metadata-ingestion/src/datahub/ingestion/source/dremio/dremio_source.py

```diff
@@ -71,6 +71,11 @@
 @capability(SourceCapability.CONTAINERS, "Enabled by default")
 @capability(SourceCapability.DATA_PROFILING, "Optionally enabled via configuration")
 @capability(SourceCapability.DESCRIPTIONS, "Enabled by default")
+@capability(
+    SourceCapability.DELETION_DETECTION,
+    "Optionally enabled via `stateful_ingestion.remove_stale_metadata`",
+    supported=True,
+)

```

Comment:
inheriting from `StatefulIngestionSourceBase` should do this automatically

can already see it in the docs here https://datahubproject.io/docs/generated/ingestion/sources/dremio/

Reply:
Let me remove then